images/nginx/rootfs/build.sh

```diff
@@ -82,33 +77,22 @@ clean-install \
   libperl-dev \
   cmake \
   util-linux \
-  lua5.1 liblua5.1-0 liblua5.1-dev \
   lmdb-utils \
   wget \
   libcurl4-openssl-dev \
   libprotobuf-dev protobuf-compiler \
   libz-dev \
   procps \
   git g++ pkgconf flex bison doxygen libyajl-dev liblmdb-dev libtool dh-autoreconf libxml2 libpcre++-dev libxml2-dev \
-  lua-cjson \
   python \
-  luarocks \
   libmaxminddb-dev \
   dumb-init \
   gdb \
   bc \
+  unzip \
+  nano \
```

Comment:
why not Vi?

Reply:
Because of the installation size "After this operation, 33.7 MB"